envs/map_refseq_ids.yml

```diff
@@ -0,0 +1,12 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - foldseek=6.29e2557
```

Comment:
I don't think `foldseek` is needed for the rule that uses this environment.

Reply:
Given that the only difference between `fetch_uniprot_metadata.yml`, `api.yml`, and `map_refseq_ids.yml` seems to be `pandas`, perhaps they could all just be one conda environment?

Reply:
@mezarque its up to you how you want to orchestrate the envs, but keeping them separate will allow them to change over time themselves without impacting the results of the others. Not really a huge deal with conda, but could reduce changes in results with containerization if you end up going that direction. Def your call tho!

Reply:
Separate seems fine!

Reply:
@mezarque: Taylor's reasoning for separate environments was exactly what I was thinking! Defs duplicated right now, but I didn't want to impose that coupling on your pipeline.